specifyweb/frontend/js_src/lib/components/Header/index.tsx

```diff
@@ -104,6 +110,11 @@ export function Header({
             ? 'dark:border-l'
             : 'dark:border-t'
         }
+        ${
+          isSideBarLight === 'matchThemeColor' && !isDarkMode
```

Comment:
could you create a variable like `isMenuDark` OR `isMenuLight` and use it in here?
that way the code would be more-self descriptive (takes a moment to realize that `isSideBarLight === 'matchThemeColor' && !isDarkMode` basically means light mode)